addons/web_editor/static/lib/summernote/src/js/summernote.js

```diff
@@ -318,7 +318,7 @@ define([
         var info = renderer.layoutInfoFromHolder($holder);
         var options = info.editor().data('options');
 
-        eventHandler.detach(info, options);
+        eventHandler.detach(info, options || {});
```

Comment:
This one was reported to me by dit but I couldn't reproduce it. Since then, carlos has encountered it in a completely different context, so I'll just remove this for now and wait for more information to surface so I can roll out a proper fix in the correct version.